code/datums/gamemode/role/syndicate.dm

```diff
@@ -24,12 +24,14 @@
 		S.set_zeroth_law("")
 		S.laws.zeroth_lock = FALSE
 		to_chat(S, "Law 0 has been purged.")
-	if(isAI(antag.current))
+	else if(isAI(antag.current))
 		var/mob/living/silicon/ai/KAI = antag.current
 		to_chat(KAI, "<b>Your laws have been changed!</b>")
 		KAI.set_zeroth_law("","")
 		KAI.laws.zeroth_lock = FALSE
 		KAI.notify_slaved()
+	else if(ishuman(antag.current))
```

Comment:
doesn't this mean that borged humans can keep their uplink?

Reply:
take_uplink is not a very intelligent proc, it just searches for an uplink anywhere on the person, so take_uplink would not do anything for borged humans anyway (there is no conceivable way they could be holding an object containing an uplink)

Reply:
It's a damn shame, because they can be made human again

Reply:
For the record, Drop() is only called by admins to remove someone's antag status, it doesn't happen when you get borged